cypress/integration/QueryParamsRedirects.js

```diff
@@ -2,28 +2,42 @@
 /* global cy */
 
 describe('Query params redirect to correct article', () => {
-  it(`User should navigate to a FAQ article with url`, () => {
+  it(`should navigate to an FAQ article with given url`, () => {
     cy.visit('/?help=/faq/RkFRQ2F0ZWdvcnk6ODk=/article/RkFRQXJ0aWNsZTo0MQ==');
   });
 
-  it(`Breadcrumbs should have length of 4`, () => {
+  it(`should have correct amount of breadcrumbs`, () => {
     cy.get('[data-cy=faq-breadcrumbs]').as('faq-breadcrumbs');
     cy
       .get('@faq-breadcrumbs')
       .children()
       .should('have.length', 3);
   });
 
-  it(`Feedback form should be present`, () => {
+  it(`should have feedback form`, () => {
     cy.get('[data-cy=faq-article-form]').as('faq-article-form');
 
     cy.get('@faq-article-form').should('exist');
   });
 
-  it(`Article title should be corect`, () => {
+  it(`should display correct article title`, () => {
     cy
       .get('.faq-article-detail')
       .find('h1')
       .contains('Cabin baggage');
   });
+
+  it('should display the root FAQ categories after clicking home breadcrumb', () => {
```

Comment:
> should display the root FAQ categories after clicking home breadcrumb

It should be `should display parent FAQ categories after clicking home breadcrumb` as tests for all parents, not only root.